filebeat/_meta/fields.common.yml

```diff
@@ -7,9 +7,14 @@
       type: keyword
       required: true
       description: >
-        The file from which the line was read. This field contains the full path to the file.
+        The file from which the line was read. This field contains the absolute path to the file.
         For example: `/var/log/system.log`.
 
+    - name: name
```

Comment:
Note sure about calling it "name" since it might ambiguous. How about "filename"?